eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,1271 @@
+          },
+          {
+            "location": "$.context.extensions.topicId",
+            "presence": "included"
+          },
+          {
+            "location": "$.verb.id",
+            "any": ["http://adlnet.gov/expapi/verbs/interacted"]
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/questionAsked",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "prefLabel": {
+          "en": "Question asked"
+        },
+        "definition": {
+          "en": "The learner was asked a question."
+        },
+        "rules": [
+          {
+            "location": "$.verb.id",
+            "any": ["http://adlnet.gov/expapi/verbs/asked"]
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.correctResponsesPattern",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.choices",
+            "presence": "included"
+          },
+          {
+            "location": "$.context.extensions.pageId",
+            "presence": "included"
+          },
+          {
+            "location": "$.context.extensions.topicId",
+            "presence": "included"
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/questionCommitted",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "prefLabel": {
+          "en": "Question committed"
+        },
+        "definition": {
+          "en": "The learner committed an answer to a question."
+        },
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.verb.id",
+            "any": ["http://adlnet.gov/expapi/verbs/answered"]
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.correctResponsesPattern",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.choices",
+            "presence": "included"
+          },
+          {
+            "location": "$.context.extensions.pageId",
+            "presence": "included"
+          },
+          {
+            "location": "$.context.extensions.topicId",
+            "presence": "included"
+          },
+          {
+            "location": "$.result.success",
+            "any": [true, false]
```

Comment:
Since these are the only two legal values, you could also write presence: included

Reply:
Good idea. I fixed that up on all the rule objects that used `location` and `any` without using `presence`.